### PR TITLE
readme: add default mode info in select section

### PR DESCRIPTION
**Pull request by ArtDu**
What has been done? Why? What problem is being solved?

I didn't forget about

- [ ] Tests
- [ ] Changelog
- [x] Documentation

Closes #???

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -892,7 +892,7 @@ where:
   * `fullscan` (`?boolean`) - if `true` then a critical log entry will be skipped
     on potentially long `select`, see [avoiding full scan](doc/select.md#avoiding-full-scan).
   * `mode` (`?string`, `read` or `write`) - if `write` is specified then `select` is
-    performed on master
+    performed on master, default value is read
```
Comment (DifferentialOrange) [inline README.md:895]:
```suggestion
    performed on master, default value is `read`
```

Comment (DifferentialOrange) [inline README.md:895]: There is also undocumented `mode` default in `get` request, let's update its description as well. `count` `mode` is already documented.

**Review by DifferentialOrange — CHANGES_REQUESTED** [review]
Thank you for you contribution! Should be ok after fixing two minor comments.

File: README.md (hunk 1 above)
Comment (ArtDu) [reply README.md]: Added to get too. Check it

**Review by DifferentialOrange — APPROVED** [review]
Thank you, LGTM!

**Comment by DifferentialOrange** [comment]
I would merge the PR after CI succeeds

**Pull request merged**